LicensePlateReader/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "package_name": "LicensePlateReader",
   "package_menu_name": "License Plate Reader",
-  "package_version": "1.0.0",
+  "package_version": "2.0.0",
```

Comment:
nevim jestli bych zvedal celou major verzi kdyz se vlastne chovani skriptu nemeni..ale asi je to jedno